### PR TITLE
seabios_order_once: add boot option order/once cases

### PR DESCRIPTION
**Pull request by ybduan**
Add seabios new cases for boot option order/once

ID: 1734201

Signed-off-by: ybduan <yduan@redhat.com>

### CODE REVIEW

**Comment by ybduan** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/2264

**Comment by ybduan** [comment]
(1/3) Host_RHEL.m7.u7.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.7.x86_64.io-github-autotest-qemu.seabios_order_once.with_order: PASS (162.98 s)
 (2/3) Host_RHEL.m7.u7.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.7.x86_64.io-github-autotest-qemu.seabios_order_once.with_once: PASS (163.76 s)
 (3/3) Host_RHEL.m7.u7.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.7.x86_64.io-github-autotest-qemu.seabios_order_once.with_order_once: PASS (165.95 s)

**Comment by XueqiangWei** [comment]
(1/2) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.seabios_order_once.with_order.q35: PASS (164.87 s)
 (2/2) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.seabios_order_once.with_order.q35: PASS (164.73 s)

File: qemu/tests/seabios_order_once.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    error_context.context("Start VM with sga bios", logging.info)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    vm.pause()
+    # Disable nic device, boot fail from nic device except user model
+    if params['nettype'] != 'user':
+        for nic in vm.virtnet:
+            process.system("ifconfig %s down" % nic.ifname)
```
Comment (luckyh) [inline qemu/tests/seabios_order_once.py]: Does `set_link` work?

File: qemu/tests/seabios_order_once.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    vm.resume()
+
+    timeout = int(params.get("login_timeout", 360))
```
Comment (luckyh) [inline qemu/tests/seabios_order_once.py]: I think the param name should be `"boot_timeout"`, and the value would be `60`

File: qemu/tests/seabios_order_once.py (hunk 1 above)
Comment (ybduan) [reply qemu/tests/seabios_order_once.py]: Change to use 'set_link', thanks!

File: qemu/tests/seabios_order_once.py (hunk 2 above)
Comment (ybduan) [reply qemu/tests/seabios_order_once.py]: Updated! 60s is too short to get info. Change to 90s. Thanks!

**Comment by ybduan** [comment]
(1/3) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.seabios_order_once.with_order: PASS (163.46 s)
 (2/3) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.seabios_order_once.with_once: PASS (163.45 s)
 (3/3) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.seabios_order_once.with_order_once: PASS (166.57 s)

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**